### DISCUSSION TITLE
Where are my files saved?

### DISCUSSION THREAD

**Discussion by nickinzon** [LeoKlaus/Paperparrot#222] [Q&A] [answered]
Hi!
I have previously been working with paperless ngx. Now I am looking for a document management system that would run as standalone application on a Mac, as running a paperless ngx server is not possible in this specific use case. Great that there is Paperparrot. I am wondering now: Where are my files being saved? I cannot find a folder in my documents folder. The application archive itself is too small for the library size the app displays. Where do my files go? I especially need to know that to make sure my library would be backed up reliably.
Thank you for a little pointer. 
Nicolas

**Comment by LeoKlaus** [comment, marked as answer]
Hey Nicolas, thank you for reaching out!

All files are stored in the documents folder of Paperparrots sandbox. On Mac, that usually is `~/Library/Group Containers/group.me.wehrfritz.paperparrot`.
Within that directory, you'll find the SQLite database(s) with all metadata and two directories `previews` and `documents`. Depending on your usage, there may be other directories and an SQLite database for custom queries as well.

Generally, if you only care about the raw files, it's enough to copy the `documents` directory to some other place. Paperparrot also has a built-in backup function, you can find it in the Settings tab at `Back Up Files and Local Database`. This will allow you to save a zip file containing all the files and directories mentioned above.

**Reply by nickinzon** [reply]
Thank you! This helps a lot.
